### PR TITLE
vscode: allow edit.find argument to be optional

### PR DESCRIPTION
**Pull request by rntz**

### CODE REVIEW

File: apps/vscode/vscode.py [hunk 1]
```diff
@@ -48,13 +48,13 @@ def filename():
 
 @ctx.action_class("edit")
 class edit_actions:
-    def find(text: str):
+    def find(text = None):
```
Comment (pokey) [inline apps/vscode/vscode.py:51]:
```suggestion
    def find(text: Optional[str] = None):
```

**Review by pokey — APPROVED** [review]
Looks good with minor comment

File: apps/vscode/vscode.py (hunk 1 above)
Comment (rntz) [reply apps/vscode/vscode.py:51]: I don't think the type signature is necessary here, as the type signature is already given when the action is declared. However, looking at actions.list() it looks like the type signature is incorrect:

```
edit.find(text: str = None)
  Open Find dialog, optionally searching for text
```

Comment (rntz) [reply apps/vscode/vscode.py:51]: Filed an issue: https://github.com/talonvoice/talon/issues/295

Comment (pokey) [reply apps/vscode/vscode.py:51]: Fair enough 🙂

**Pull request merged**